src/main/java/io/temporal/internal/replay/ReplayWorkflowContext.java

```diff
@@ -104,39 +104,39 @@
    * @param parameters An object which encapsulates all the information required to schedule an
    *     activity for execution
    * @param callback Callback that is called upon activity completion or failure.
-   * @return cancellation handle. Invoke {@link Consumer#accept(Object)} to cancel activity task.
+   * @return cancellation handle. Invoke {@link Functions.Proc1#apply(Object)} to cancel activity
+   *     task.
    */
-  Consumer<Exception> scheduleActivityTask(
-      ExecuteActivityParameters parameters, BiConsumer<Optional<Payloads>, Exception> callback);
+  Functions.Proc1<Exception> scheduleActivityTask(
+      ExecuteActivityParameters parameters, Functions.Proc2<Optional<Payloads>, Failure> callback);
 
-  Consumer<Exception> scheduleLocalActivityTask(
+  Functions.Proc scheduleLocalActivityTask(
       ExecuteLocalActivityParameters parameters,
-      BiConsumer<Optional<Payloads>, Exception> callback);
+      Functions.Proc2<Optional<Payloads>, Failure> callback);
 
   /**
    * Start child workflow.
    *
    * @param parameters An object which encapsulates all the information required to schedule a child
    *     workflow for execution
    * @param callback Callback that is called upon child workflow completion or failure.
-   * @return cancellation handle. Invoke {@link Consumer#accept(Object)} to cancel activity task.
+   * @return cancellation handle. Invoke {@link Functions.Proc1#apply(Object)} to cancel activity
+   *     task.
    */
-  Consumer<Exception> startChildWorkflow(
+  Functions.Proc1<Exception> startChildWorkflow(
       StartChildWorkflowExecutionParameters parameters,
-      Consumer<WorkflowExecution> executionCallback,
-      BiConsumer<Optional<Payloads>, Exception> callback);
+      Functions.Proc1<WorkflowExecution> executionCallback,
+      Functions.Proc2<Optional<Payloads>, Exception> callback);
 
-  Consumer<Exception> signalWorkflowExecution(
+  Functions.Proc1<Exception> signalExternalWorkflowExecution(
```

Comment:
Curious why some of these methods have full comments (such as startChildWorkflow), but then some like this one (signalExternalWorkflowExecution) don't